users/Tal296/weighted_average.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+
+def compute_weighted_average(x: List[float],
+                             w: List[float])-> float:
+    try:
+        return sum([x1 * w1 for x1, w1 in zip(x, w)]) / sum(w)
+    except:
+        print("Cannot divide by zero")
```

Comment:
I really like this solution!
I think it's a good idea to crush the run.

But one thing:
Usually catching general exceptions it's bad practice.
(sometimes unexpected exceptions can be raised and the code in the except block won't treat them well)
I would replace it with `except ZeroDivisionError:` or with:
```suggestion
    assert sum(w) !=  0, "Sum of weights is zero!"
    return sum([x1 * w1 for x1, w1 in zip(x, w)]) / sum(w)
```

Reply:
Thanks for the review.